src/main/java/moze_intel/projecte/gameObjs/items/tools/PEToolBase.java

```diff
@@ -574,7 +574,7 @@ protected void mineOreVeinsInAOE(ItemStack stack, EntityPlayer player) {
 		for (BlockPos pos : WorldHelper.getPositionsFromBox(box))
 		{
 			IBlockState state = world.getBlockState(pos);
-			if (ItemHelper.isOre(state) && state.getBlockHardness(player.worldObj, pos) != -1 && (canHarvestBlock(state, stack) || ForgeHooks.canToolHarvestBlock(world, pos, stack)))
+			if (state != Blocks.AIR.getDefaultState() && ItemHelper.isOre(state) && state.getBlockHardness(player.worldObj, pos) != -1 && (canHarvestBlock(state, stack) || ForgeHooks.canToolHarvestBlock(world, pos, stack)))
```

Comment:
Don't know anymore. Might not be needed anymore, because I added the null check to the oredict function. 

The isOre function did crash for air blocks.